django_multitenant/django_multitenant.py

```diff
@@ -107,6 +107,81 @@ def _do_update(self, base_qs, using, pk_val, values, update_fields, forced_updat
     class Meta:
         abstract = True
 
+
+class TenantForeignKey(models.ForeignKey):
+    '''
+    Should be used in place of models.ForeignKey for all foreign key relationships to
+    subclasses of TenantModel.
+
+    Adds additional clause to JOINs over this relation to include tenant_id in the JOIN
+    on the TenantModel.
+
+    Adds clause to forward accesses through this field to include tenant_id in the
+    TenantModel lookup.
+    '''
+
+    # Override
+    def get_extra_descriptor_filter(self, instance):
+        """
+        Return an extra filter condition for related object fetching when
+        user does 'instance.fieldname', that is the extra filter is used in
+        the descriptor of the field.
+
+        The filter should be either a dict usable in .filter(**kwargs) call or
+        a Q-object. The condition will be ANDed together with the relation's
+        joining columns.
+
+        A parallel method is get_extra_restriction() which is used in
+        JOIN and subquery conditions.
+        """
+        current_tenant = get_current_tenant()
+        if current_tenant:
+            return {instance.__class__.tenant_id: current_tenant.id}
+        else:
+            logger.warn('TenantForeignKey field %s.%s on instance "%s" '
```

Comment:
When I go through this code path I get this error:
```
NameError: name 'logger' is not defined
```
Just adding something as below to the **django_multitennat.py** file would fix it.
```
import logging
logger = logging.getLogger(__name__)
```

Reply:
Ah yup, thanks